src/main/java/com/minecraftabnormals/abnormals_core/core/util/DataUtil.java

```diff
@@ -72,4 +74,18 @@ public static boolean matchesKeys(ResourceLocation loc, RegistryKey<?>... keys)
 				return true;
 		return false;
 	}
+
+	/**
+	 * <p>Registers a {@link IDispenseItemBehavior} that will perform the new behavior if the condition is met
+	 * and the behavior that was already in the registry if not.</p>
+	 * This works even if multiple mods add new behavior to the same item.
+	 *
+	 * @author abigailfails
+	 * */
+	public static void registerConditionalDispenseBehavior(Item item, BiPredicate<IBlockSource, ItemStack> condition, IDispenseItemBehavior newBehavior) {
```

Comment:
`registerAlternativeDispenseBehavior` would be a better method name, in my opinion.

Reply:
Changed in latest commit.